Services/Store/ModsenOnlineStore.Store.Infrastructure/Data/OrderRepository.cs

```diff
@@ -13,9 +13,11 @@ public OrderRepository(DataContext context)
             this.context = context;
         }
 
-        public async Task<List<Order>> GetAllOrders()
+        public async Task<List<Order>> GetAllOrders(int pageNumber, int pageSize)
         {
-            return await context.Orders.AsNoTracking().ToListAsync();
+            var orders = await context.Orders.AsNoTracking().ToListAsync();
```

Comment:
Insert `ToPagedList()` into chain of call, so entity framework can do pagination directly in database

Reply:
Is it mean that i need to make ToPagedList() an extension method?

Reply:
yes